cmd/kops/update_cluster.go

```diff
@@ -172,6 +175,20 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 		glog.Infof("Using SSH public key: %v\n", c.SSHPublicKey)
 	}
 
+	var phase cloudup.Phase
```

Comment:
Yes, the majority of user that would be interested in this would be interesting with this with yaml support. We can merge and move lower if you like.